proactive-support-bot.py

```diff
@@ -56,6 +57,13 @@ def handle_fileuploadtest(client, event):
         channels=channel,
         content="Hello, World")
 
+def handle_cluster(client, event):
+    channel = event['event']['channel']
+    cluster = event['event'].get('text').split()[-1]
+    account = ocm.cluster_to_account(os.environ['PYOCM_SECRETpyocm'], cluster)
```

Comment:
`PYOCM_SECRETpyocm` -> `PYOCM_SECRET`?

Reply:
Or `PYOCM_TOKEN`?

Reply:
It is the way the secret got added as the environment variable. I'm happy to modify it. hang tight and I'll go update it for the pod

Reply:
and maybe pull it out as a module global:

```python
ocm_token = os.environ['OCM_TOKEN']
```
and then here:
```python
account = ocm.cluster_account(token=ocm_token, cluster=cluster)
```

Reply:
Both are great points, handled in recent commit.